opv_directorymanager/storage_service/http.py

```diff
@@ -69,6 +79,23 @@ def file_send(name):
             p = os.path.join(self.__path, "/".join(temp))
             if os.path.isdir(p):
                 return json.dumps(os.listdir(p))
+
+            if magic.from_file(p, mime=True).split("/")[0] == "image" and (request.args.get('scale') is not None or request.args.get('width') is not None or request.args.get('height') is not None):
+                img = Image.open(p)
+                size = [0, 0]
+                if request.args.get('scale', type=int) is not None:
+                    scale = request.args.get('scale', type=int)
+                    size = [img.size[0]/scale, img.size[1]/scale]
+                elif request.args.get('width', type=int) is not None:
+                    size = [request.args.get('width', type=int), img.size[1]]
```

Comment:
Wrong ratio, no need to handle it as thumbnail PIL will compute it. As it's stated in the documentation "This method modifies the image to contain a thumbnail version of itself, no larger than the given size. " https://pillow.readthedocs.io/en/3.1.x/reference/Image.html#PIL.Image.Image.thumbnail